<ui version="4.0" >
 <class>CookiesExceptionsDialog</class>
 <widget class="QDialog" name="CookiesExceptionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cookie Exceptions</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="newExceptionGroupBox" >
     <property name="title" >
      <string>New Exception</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Domain:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="domainLineEdit" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>81</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="blockButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Block</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="allowForSessionButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Allow For Session</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="allowButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Allow</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ExceptionsGroupBox" >
     <property name="title" >
      <string>Exceptions</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>252</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" >
       <widget class="SearchLineEdit" name="search" />
      </item>
      <item row="1" column="0" colspan="4" >
       <widget class="EditTableView" name="exceptionTable" />
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="removeButton" >
        <property name="text" >
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="removeAllButton" >
        <property name="text" >
         <string>Remove &amp;All</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>searchlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EditTableView</class>
   <extends>QTableView</extends>
   <header>edittableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CookiesExceptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>381</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel" >
     <x>336</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
